.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+
+python:
+  - 3.6
+
+install: "pip install -r requirements.txt"
+
+before_script: ablog build
```

Comment:
Uh, my mistake. I thought ablog won't build the rst files without `ablog build` , but `make html` seems enough. I'll change the yml file in the next commit.